From c0f5465dd4a1fbd2e9cfc277d1c0b39aa99c182c Mon Sep 17 00:00:00 2001
From: Abe Lopez <abe.lopez@onsemi.com>
Date: Mon, 18 Dec 2017 10:45:25 -0800
Subject: [PATCH] Remove AllocedDict constructor.

This isn't used by us and gives errors on compile.
---
 Fleece/FleeceCpp.hh | 19 ++++++++++---------
 1 file changed, 10 insertions(+), 9 deletions(-)

diff --git a/Fleece/FleeceCpp.hh b/Fleece/FleeceCpp.hh
index 4238af6..589be91 100644
--- a/Fleece/FleeceCpp.hh
+++ b/Fleece/FleeceCpp.hh
@@ -41,13 +41,13 @@ namespace fleeceapi {
 
     static inline bool operator== (FLSliceResult sr, FLSlice s) {return (FLSlice)sr == s;}
     static inline bool operator!= (FLSliceResult sr, FLSlice s) {return !(sr ==s);}
-    
+
 
     class Value {
     public:
         static Value fromData(FLSlice data)          {return Value(FLValue_FromData(data));}
         static Value fromTrustedData(FLSlice data)   {return Value(FLValue_FromTrustedData(data));}
-        
+
         Value()                                         { }
         Value(FLValue v)                                :_val(v) { }
         operator FLValue const ()                       {return _val;}
@@ -123,8 +123,8 @@ namespace fleeceapi {
 
         FLKeyPath _path;
     };
-    
-    
+
+
     class Array : public Value {
     public:
         Array()                                         :Value() { }
@@ -247,10 +247,11 @@ namespace fleeceapi {
         explicit AllocedDict(fleece::slice s)
         :AllocedDict(alloc_slice(s)) { }
 
-        AllocedDict(const AllocedDict &d)
-        :Dict(d)
-        ,alloc_slice(d)
-        { }
+        // ALopez
+        // AllocedDict(const AllocedDict &d)
+        // :Dict(d)
+        // ,alloc_slice(d)
+        // { }
 
         const alloc_slice& data() const                 {return *this;}
     };
@@ -269,7 +270,7 @@ namespace fleeceapi {
         { }
 
         void release()                                  {_enc = nullptr;}
-        
+
         ~Encoder()                                      {FLEncoder_Free(_enc);}
 
         void setSharedKeys(FLSharedKeys sk)             {FLEncoder_SetSharedKeys(_enc, sk);}
-- 
2.7.4

